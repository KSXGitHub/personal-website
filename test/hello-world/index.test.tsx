import HelloWorld from 'hello-world'

it('matches snapshot', () => {
  expect(HelloWorld({ message: 'hello' })).toMatchSnapshot()
})
