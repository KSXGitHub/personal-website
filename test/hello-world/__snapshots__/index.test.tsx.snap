// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<main
  style={
    Object {
      "display": "block",
      "height": "100%",
      "margin": "0px",
      "padding": "0px",
      "width": "100%",
    }
  }
>
  <h1
    style={
      Object {
        "display": "block",
        "fontFamily": "sans-serif",
        "height": "100%",
        "textAlign": "center",
        "verticalAlign": "middle",
        "width": "100%",
      }
    }
  >
    hello
  </h1>
</main>
`;
